scrapers/BoyFun.yml

```diff
@@ -0,0 +1,31 @@
+name: BoyFun
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.boyfun.com/video
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: 
+        selector: //span[@class="title"]
+      Image: //video/@poster
+      Date:
+        selector: //span[@class="date"]/span[@class="content"]
+        postProcess:
+          - replace:
+              - regex: (st|[nr]d|th) 
+                with: "" 
+          - parseDate: Jan 2, 2006
+      #Details: //div[@class="heading"]/following-sibling::p[1]
+      Details: 
+        selector: //div[@class="heading"]/following-sibling::text()
```

Comment:
`//div[@class='content-information-description']/p/text()` also works. Any reason to not go with the more precise selector?

Reply:
> `//div[@class='content-information-description']/p/text()` also works. Any reason to not go with the more precise selector?

No specific reason, except that I probably missed the obvious.